webapp/lib/charts.dart

```diff
@@ -159,6 +161,8 @@ class DailyTimeseriesLineChartView {
         ..clear()
         ..addAll(timeseriesPoints);
     }
+    var isOneDayFiltered = view.ChartFiltersView().selectedPeriodFilter == 'ChartPeriodFilters.days1';
+    chart.options.scales.xAxes[0].time = (new chartjs.TimeScale(unit: (isOneDayFiltered ? 'hour' : 'day')));
```

Comment:
I'd rather we didn't create a dependency on view.dart like this. Can we have the timescale unit as an optional parameter for the `updateChart` method instead, which is being computed in `view.dart` by the caller?

Reply:
Yes. I've changed it. Much cleaner